business_requirement_project/__openerp__.py

```diff
@@ -5,7 +5,7 @@
     'name': 'Business Requirement - Project',
     'category': 'Business Requirements Management',
     'summary': 'Business Requirement - Project',
-    'version': '8.0.2.0.1',
+    'version': '8.0.4.0.1',
```

Comment:
@elicoidal
Before I create the PR with this commit: https://github.com/victormartinelicocorp/project/commit/385b4af2e82fad86f7f0b5abc2217eae139060fe
And later I create a new branch based on the  `version': '8.0.3.0.1'`  and I modify again the module. I know that maybe I could merged the two task in one because I modify the same module at the same method. 
Do you want me to fix the version number?
cc @elicoidal